app/src/main/java/com/elifut/activity/MatchProgressActivity.java

```diff
@@ -242,6 +248,24 @@ private void appendEvent(@DrawableRes int icon, String text, int gravity) {
     eventsScrollView.smoothScrollTo(0, 0);
   }
 
+  private void setGameSpeed(int gameSpeed) {
+    userPreferences.gameSpeedPreference().set(gameSpeed);
+    stopTimer();
+    startTimer();
+  }
+
+  @OnClick(R.id.fab_backward) void onBackward() {
+    int currSpeed = userPreferences.gameSpeed();
+    currSpeed/=2;
+    setGameSpeed(Math.max(MIN_GAME_SPEED, currSpeed));
+  }
+
+  @OnClick(R.id.fab_fast_forward) void onFastForward() {
+    int currSpeed = userPreferences.gameSpeed();
```

Comment:
should we hide/disable the fast forward/backward buttons once we reach the max/min speeds?